cognee/modules/data/extraction/knowledge_graph/extract_content_graph.py

```
@@ -2,12 +2,27 @@
+from dotenv import load_dotenv
+import os
 
 
 async def extract_content_graph(content: str, response_model: Type[BaseModel]):
     llm_client = get_llm_client()
+    
+# At the top of the file, after imports
+load_dotenv()
 
-    system_prompt = render_prompt("generate_graph_prompt.txt", {})
+async def extract_content_graph(content: str, response_model: Type[BaseModel]):
```

Ruff (0.8.2)

Redefinition of unused `extract_content_graph` from line 9

(F811)